0.1.0/lib.typ

```diff
@@ -49,22 +52,22 @@
   show heading.where(
     level: 1
   ): it => {
-    pagebreak()
-    text(size: 26pt, it)
+    pagebreak(weak: true)
```

Comment:
Somewhere below is a `show pagebreak(weak: true)` rule that can be moved up to do this cleaner